UnityProject/Assets/Textures/objects/machines/service_bell/service_bell.asset

```diff
@@ -0,0 +1,20 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!114 &11400000
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 0}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: 9d1de2691495b304e9843ac3335b626d, type: 3}
+  m_Name: service_bell
+  m_EditorClassIdentifier: 
+  Variance:
+  - Frames:
+    - sprite: {fileID: 21300000, guid: 897babe988b324547a82d30795355c76, type: 3}
+      secondDelay: 0
+  IsPalette: 0
+  setID: 0
```

Comment:
It looks like the sprite ID wasn't set automatically after all. I left it empty on the advice of Bod9001.

How do I fix this?

Reply:
leave empty and reimport the asset, or so I understood from what Bod said

Reply:
If I leave it empty, the value becomes `0`. And it seems that reimport doesn't do anything useful to solve the problem.

Or maybe I'm doing something wrong?

1. Openning `Service_ball (Sprite Data SO)`.
2. Setting the Set ID value to blank(which becomes zero).
3. Right-clicking on the `service_ball` selected from the bottom.
4. Clicking reimport.

Reply:
then try deleting the sprite, meta and so. Do the process of importing the image into the project again.

Reply:
Looks like it's fixed. Thanks

Reply:
wonderful. Waiting for tests then I can merge. Thanks for the contribution!